docs/rpc/rpc-spans.md

```diff
@@ -192,23 +284,23 @@ client and server spans SHOULD be created. In case of unary calls only one sent
 and one received message will be recorded for both client and server spans.
 
 <!-- semconv rpc.message(full) -->
-The event name MUST be `rpc.message`.
+The event name MUST be `rpc.message`
```

Comment:
No.  At some point I want to migrate this to a JINJA template, but this would break --dry-run for various reasons.

I'll add a fix for this in weaver.

Reply:
https://github.com/open-telemetry/weaver/issues/126